packages/redis/rollup.config.ts

```diff
@@ -19,6 +19,6 @@ export default {
       sourcemap: true
     }
   ],
-  external: ['@joshdb/core', '@joshdb/serialize', '@sapphire/utilities', 'redis', 'uuid'],
+  external: [],
```

Comment:
This needs resolving. Run `yarn workspace @joshdb/redis run build` to see which packages to add to the array.